composer.json

```diff
@@ -22,6 +22,7 @@
     "symfony/framework-bundle": "~2.8|~3.0",
     "symfony/browser-kit": "~2.8|~3.0",
     "symfony/expression-language": "~2.8|~3.0",
+    "symfony/console": "~2.8|~3.0",
```

Comment:
After some tests, it is not what we want: prepending the configuration counts like if it was from the user (it is merged with the other values, not overrided).
So instead, the default value is just changed in the Configuration class.